ci/Jenkinsfile.tests-e2e

```diff
@@ -158,8 +158,7 @@ pipeline {
           $class:            'Xvfb',
           autoDisplayName:   true,
           parallelBuild:     true,
-          screen:            '2560x1440x24',
-          additionalOptions: '-dpi 1'
+          screen:            '1600x1200x16',
```

Comment:
Out default window size is:
```qml
    minimumWidth: 1200
    minimumHeight: 700
```

Maybe we could follow that here as well

Reply:
@jakubgs was suggesting to try 1280x1024